keps/sig-node/compute-device-assignment.md

```diff
@@ -164,6 +184,7 @@ Beta:
 
 ## Implementation History
 
+- 2020-08-XX: KEP extended with ListAndWatch function
```

Comment:
We would need to update the date here and change ListAndWatch to Watch. Just adding this as a placeholder so we don't forget later